content/en/security/code_security/static_analysis/_index.md

```diff
@@ -2,61 +2,49 @@
 title: Static Analysis (SAST)
 description: Learn about Datadog Static Analysis to scan code for quality issues and security vulnerabilities before your code reaches production.
 aliases:
-- /continuous_integration/static_analysis
-- /static_analysis
+- /code_analysis/static_analysis
 is_beta: false
-further_reading:
-- link: "https://www.datadoghq.com/blog/monitor-ci-pipelines/"
-  tag: "Blog"
-  text: "Monitor all your CI pipelines with Datadog"
-- link: "/integrations/guide/source-code-integration/"
-  tag: "Documentation"
-  text: "Learn about the Source Code Integration"
 algolia:
   tags: ['static analysis', 'datadog static analysis', 'code quality', 'SAST']
 ---
 
-{{< callout url="#" btn_hidden="true" header="Join the Preview!" >}}
-Code Analysis is in Preview.
-{{< /callout >}}
-
 {{% site-region region="gov" %}}
 <div class="alert alert-danger">
-    Code Analysis is not available for the {{< region-param key="dd_site_name" >}} site.
+    Code Security is not available for the {{< region-param key="dd_site_name" >}} site.
 </div>
 {{% /site-region %}}
 
 
 ## Overview
 
-Static Analysis (SAST) is a clear-box software testing technique that analyzes a program's pre-production code without the need to execute the program, meaning that the program is static because it isn't running. 
+Static Analysis is Datadog's Static Application Security Testing (SAST) capability. SAST is a clear-box software testing technique that analyzes a program's pre-production code without the need to execute the program, meaning that the program is static because it isn't running. 
```

Comment:
```suggestion
Static Code Analysis is Datadog's Static Application Security Testing (SAST) capability. SAST is a clear-box software testing technique that analyzes a program's pre-production code without the need to execute the program, meaning that the program is static because it isn't running. 
```